extension/intents/window/window.toml

```diff
@@ -10,3 +10,19 @@ description = "Open downloads"
 match = """
   (show | open |) downloads (menu |)
 """
+
+[window.combine]
+description = "Combine all windows"
+match = """
+  combine all windows (into |) (one |) (window |)
+  combine tabs (into |) (one |) window
+"""
+
+[[window.combine.example]]
+phrase = "combine all windows"
+
+[[window.combine.example]]
+phrase = "combine all windows into one"
+
+[[window.combine.example]]
+phrase = "combine tabs into one window"
```

Comment:
We only need one of these examples to actually display to the user, so the other examples should have `test = true` (they'll still be tested, but not displayed).

Reply:
Okay, I see now! I also was not sure whether I need to add more examples or matchings and decided that this is sufficient